include/pf-applications/lac/dynamic_block_vector.h

```diff
@@ -400,6 +400,8 @@ namespace dealii
           std::swap(this->size_, V.size_);
         }
 
+        static constexpr unsigned int communication_block_size = 0;
```

Comment:
@vovannikov This is needed to disallow `MatrixFree` to overlap the communication of multiple blocks.